requirements.txt

```diff
@@ -1,4 +1,4 @@
-psycopg2
+psycopg2==2.7.4
```

Comment:
Is there a problem with newer versions?

Reply:
No, it can be psycopg2>=2.7.4

Reply:
This is the latest version available, released on Feb 6.